internal/rollup/rollup_bundle.bzl

```diff
@@ -30,6 +51,7 @@ def _rollup_bundle(ctx):
           "TMPL_workspace_name": ctx.workspace_name,
           "TMPL_build_file_dirname": buildFileDirname,
           "TMPL_label_name": ctx.label.name,
+          "TMPL_module_mappings": str(mappings),
```

Comment:
instead of just stringifying, could you use the same serialization as https://github.com/bazelbuild/rules_nodejs/blob/master/internal/node.bzl#L46

Reply:
do you think stringifying could fail in some cases?